Services/TMS/TrainingSearch/classes/class.ilTrainingSearchGUI.php

```diff
@@ -27,9 +28,13 @@ class ilTrainingSearchGUI {
 	protected $g_ctrl;
 
 	/**
-	 * @var ilObjUser
+	 * User id of user who going to get booked.
+	 * Initial the current ilUser.
+	 * Might be changed to other is current is allowd to book for other
+	 *
```

Comment:
UserId of the user that is going to be booked. Initially set to current ilUser.
This might be changed, if the current user is allowed to book for others.